docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller_test.go

```diff
@@ -0,0 +1,200 @@
+/*
+We can create the tests in a separate file, cronjob_controller_test.go.
+Keeping consistent with the Gingko usage in our autogenerated suite_test.go code, we’ll use Gingko to write our tests.
```

Comment:
Is it really crucial info over how to do it?  I think we can remove it. To try to have less verbose. 
```suggestion
```

Reply:
I can remove this! It's not the most necessary, it's just additional context for the testing framework also used in this file.